composer.json

```
@@ -6,7 +6,8 @@
         "php": ">=7.1.0",
         "silverstripe/recipe-plugin": "^1.2",
         "silverstripe/recipe-cms": "4.5.x-dev",
-        "silverstripe-themes/simple": "~3.2.0"
+        "silverstripe-themes/simple": "~3.2.0",
+        "silverstripe/login-forms": "^4.0.x-dev"
```

```suggestion
        "silverstripe/login-forms": "4.0.x-dev"
```

I don't think the previous version works